### PR TITLE
Polidea update - direct links to Ecosystem tabs

### PR DESCRIPTION
**Pull request by wbartosz**
This pull request includes a requested change (https://github.com/istio/istio.io/issues/9466) to access a specific tab based on the url hash.
It also gives ability to stop browser from remembering the latest tab selected (since it reuses the tabset component, it is useful not to remember the latest tab selected). 

Implemented on the Ecosystem page.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[X] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/about/ecosystem/index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ doc_type: about
 
     {{< tab
         name="pro services"
-        category-value="pro_services"
+        category-value="pro-services"
```
Comment (craigbox) [inline content/en/about/ecosystem/index.md:27]: just "services" please
```suggestion
        category-value="services"
```

**Review by craigbox — DISMISSED** [review]

File: content/en/about/ecosystem/index.md (hunk 1 above)
Comment (wbartosz) [reply content/en/about/ecosystem/index.md]: Done

**Review by ericvn — APPROVED** [review]

**Comment by davidhauck** [comment]
lgtm

**Pull request merged**